egs2/lrs3/asr1/conf/slurm.conf

```diff
@@ -0,0 +1,17 @@
+# Default configuration                                                                                                                                                        
+command sbatch --export=PATH
+option name=* --job-name $0
+default time=48:00:00
+option time=* --time $0
+option mem=* --mem-per-cpu $0
+option mem=0
+option num_threads=* --cpus-per-task $0
+option num_threads=1 --cpus-per-task 1
+option num_nodes=* --nodes $0
+default gpu=0
+option gpu=0 -p RM-shared
+option gpu=8 -p GPU --gres=gpu:v100-32:8 -c 32
+option gpu=1 -p GPU-shared --gres=gpu:v100-32:1 -c 5
+option gpu=* -p GPU-shared --gres=gpu:v100-32:$0 -c $0  # Recommend allocating more CPU than, or equal to the number of GPU
+# note: the --max-jobs-run option is supported as a special case
+# by slurm.pl and you don't have to handle it in the config file.
```

Comment:
This config is a bit different from the default slurm.conf 
Update this to the default